src/routes/(app)/admin[[hash=admin_hash]]/arm/+page.svelte

```diff
@@ -209,6 +214,13 @@
 						</option>
 					{/each}
 				</select>
+				<input
+					class="form-checkbox place-self-center"
+					type="checkbox"
+					name="hasPosOptions"
+					bind:checked={user.hasPosOptions}
+					disabled={data.roles.find((rol) => rol._id === user.roleId)?.hasPosOptions}
+				/>
```

Comment:
If the user role has POS access, the user checkbox will be disabled (as expected), but unchecked.

It would be best if the checkbox was checked and disabled; a tooltip could be displayed on hover explaining the user this user has POS access because of their role.